builders/Dockerfile.centos7

```diff
@@ -3,6 +3,13 @@ FROM centos:centos7
 RUN yum -y update \
   && yum -y install clang gcc gcc-c++ make wget
 
+# Obtain dependencies for cross compiling
+# The only prebuilt glibc was done by third party Computational
+RUN yum -y install epel-release 
+RUN yum -y install gcc-aarch64-linux-gnu gcc-c++-aarch64-linux-gnu
+RUN yum install -y http://springdale.princeton.edu/data/springdale/7/x86_64/os/Computational/glibc-aarch64-linux-gnu-2.24-2.sdl7.2.noarch.rpm
+RUN yum install -y http://springdale.princeton.edu/data/springdale/7/x86_64/os/Computational/glibc-aarch64-linux-gnu-devel-2.24-2.sdl7.2.noarch.rpm 
```

Comment:
Nice.

I tried to reduce the number of `RUN`s in order to avoid unnecessary Docker layers and image size. This is the reason for the long `&&` chains in the file. But this is just a 💅

Reply:
I can do that.